lib/Cargo.toml

```diff
@@ -12,6 +12,7 @@ env_logger = "0.8.4"
 directories = "5.0.1" # For Environment to find platform-specific config location
 smallvec = "1.10.0"
 im = "15.1.0"
+bitset = "0.1.2"
```

Comment:
bitset's implementation isn't very optimal, but it'll do for now.  If it ever shows up as a bottleneck we have a lot of experience writing crazy-fast bit-set operations ;-)